### PR TITLE
BOLT 7: readability fixup

### LINKED ISSUE #443: BOLT 7: difference `final node` and `endpoint node`
There are two node in [BOLT 7 Rebroadcasting Requirements](https://github.com/lightningnetwork/lightning-rfc/blob/5c11b6a4022b75fc7717133081eab6f34d7d56a6/07-routing-gossip.md#requirements-5) :
  * `final node`
  * `endpoint node`

What are the differences between the two?

I guess below in BOLT 7:
* `origin node`
  * message creation node and broadcast it
* `final node`
  * message receiving node and not forward
* `endpoint node`
  * message receiving node and maybe forward other nodes

### PR DESCRIPTION
**Pull request by niftynei**
make it a bit easier to read.

fixes #443

### CODE REVIEW

File: 07-routing-gossip.md [hunk 1]
```diff
@@ -540,7 +540,7 @@ An endpoint node:
 
 ### Requirements
 
-The final node:
+A message receiving node:
```
Comment (rustyrussell) [inline 07-routing-gossip.md]: A receiving node?

File: 07-routing-gossip.md [hunk 2]
```diff
@@ -553,7 +553,7 @@ The final node:
         - Note: this will later allow the final node to rebuild the announcement
         for its peers.
 
-An endpoint node:
+A node:
```
Comment (rustyrussell) [inline 07-routing-gossip.md]: Yes; in fact, every occurance of `final node` or `endpoint node` in this BOLT are wrong...

**Comment by rustyrussell** [comment]
Assigned to @sstone for final review; I believe he had questions, otherwise @sstone please commit as per minutes:
https://docs.google.com/document/d/1GaH4ERsFxnVRceW-hG_-Ul5TOjRv40yB2S3-CEe5n9s/edit#

File: 07-routing-gossip.md [hunk 3]
```diff
@@ -753,15 +683,89 @@ one.  If there's no `channel_update` then it is not sent at all, which is most
...
+  - SHOULD flush outgoing gossip messages once every 60 seconds, independently of
+  the arrival times of the messages.
+    - Note: this results in staggered announcements that are unique (not
+    duplicated).
+  - MAY re-announce its channels regularly.
+    - Note: this is discouraged, in order to keep the resource requirements low.
+  - upon connection establishment:
+    - SHOULD send all `channel_announcement` messages, followed by the latest
+    `node_announcement` AND `channel_update` messages.
+
```
Comment (sstone) [inline 07-routing-gossip.md]: I think this section ("upon connection establishment") should just be removed: as is it is wrong, and what nodes should do when they receive the `init` message is already described in this document.

Comment (niftynei) [reply 07-routing-gossip.md]: hi @sstone this section has not been modified in this PR, it's merely been moved such that the section ordering matches the ordering listed in the Table of Contents for this BOLT spec. if we're going to delete it, we should merge this PR in first (to fix the ordering) and then edit the text to be correct.

**Pull request merged**